src/app/shared/services/apis/gb3/gb3-print.service.ts

```diff
@@ -177,6 +180,35 @@ export class Gb3PrintService extends Gb3ApiService {
   }
 
   private createGb2WmsPrintItem(activeMapItem: ActiveMapItem, gb2WmsSettings: Gb2WmsSettings): PrintMapItem {
```

Comment:
Please add some unit tests for this; or extend existing ones to reach the coverage goal.

Reply:
90% coverage reached! Thanks ❤️